model/metrics/gen-ai.yaml

```diff
@@ -43,3 +43,19 @@ groups:
           The `error.type` SHOULD match the error code returned by the Generative AI provider or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+  - id: metric.gen_ai.server.latency.time_per_output_token
```

Comment:
do we need an `error_type` attribute here? Or this metrics only for successful replies?

Reply:
These metrics are not per request and are cumulative for all successful requests only. So, I don't think `error_type` is needed here. Let me know if that makes sense.

Reply:
ok, perhaps add this assumption into the metric description? Something like 'Time per output token generated for successful responses'

Reply:
Makes sense. Added.